pandas/core/frame.py

```diff
@@ -105,7 +105,9 @@
     optional_by="""
         by : str or list of str
             Name or list of names which refer to the axis items.""",
-    versionadded_to_excel='')
+    versionadded_to_excel='',
+    versionadded_melt='\n.. versionadded:: 0.20.0\n',
```

Comment:
this needs extra spaces (you can check `pd.DataFrame.melt?` if you are on this branch)